packages/workflow/src/TelemetryHelpers.ts

```diff
@@ -525,3 +526,59 @@ export const userInInstanceRanOutOfFreeAiCredits = (runData: IRun): boolean => {
 
 	return false;
 };
+
+export type FromAICount = {
+	aiNodeCount: number;
+	aiToolCount: number;
+	fromAIOverrideCount: number;
+	fromAIExpressionCount: number;
+};
+
+export function makeAIMetrics(nodes: INode[], nodeTypes: INodeTypes): FromAICount | {} {
```

Comment:
`make metrics` sounds a little weird in my head. `Resolve` might be clearer. 
```suggestion
export function resolveAIMetrics(nodes: INode[], nodeTypes: INodeTypes): FromAICount | {} {
```